src/PrestaShopBundle/Resources/views/Admin/Sell/Order/Order/Blocks/View/messages.html.twig

```diff
@@ -78,9 +78,9 @@
       {% endfor %}
     </div>
 
-    <div class="form-group row">
+    <div class="form-group row configure">
       <div class="col-sm">
-        <a href="{{ path('admin_order_messages_index') }}" class="mb-1">
+        <a href="{{ path('admin_order_messages_index') }}" class="configure-link">
           {{ 'Configure predefined messages'|trans({}, 'Admin.Orderscustomers.Feature') }}
```

Comment:
I have a slight doubt. @MatShir, did we change this wording into something shorter during the first reviews? It kind of rings a bell to me but I can't put my finger on it...
🤔

Reply:
This PR is coming from 1.7.7.x, it means that this wording has been merged on 1.7.7.x, maybe the change has been forgot in the past or hasn't been done

Reply:
Okay, well, let's keep it as is!